src/components/AvatarHeader/AvatarHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useParams, useHistory } from 'react-router';
+import { Box, createStyles, Theme } from '@material-ui/core';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { useQuery } from '@apollo/react-hooks';
+import Avatar from 'components/Avatar';
+import AVATAR_HEADER from '../../graphql/avatar-header.graphql';
```

Comment:
`../../` - since we can use absolute paths, this should be removed for all imports